src/PowerSimulations.jl

```diff
@@ -153,26 +163,29 @@ JuMP.Model(optimizer::Nothing; kwargs...) = JuMP.Model(kwargs...)
 ################################################################################
```

Comment:
I'm not sure why a user would ever want to pass an optimizer as nothing. Can you clarify why this is needed by adding a comment about why this is being done?

Also, move 

```julia
JuMP.Model(optimizer::Nothing; kwargs...) = JuMP.Model(kwargs...)
```

to the end of the file when adding a comment about why this method overloading is required.